module/vision/BallDetector/src/BallDetector.cpp

```diff
@@ -44,6 +44,7 @@ namespace module::vision {
     using message::vision::GreenHorizon;
 
     using utility::math::coordinates::cartesianToSpherical;
+    using utility::math::coordinates::inverseDistanceCartesianToSpherical;
```

Comment:
I feel like this name is misleading, it implies the cartesian is the inverse distance.
Can it be renamed to `cartesianToInverseDistanceSpherical` or similar?